dtschema/schemas/gpio/gpio.yaml

```diff
@@ -35,7 +35,6 @@ properties:
 
 dependencies:
   gpio-controller: ['#gpio-cells']
-  '#gpio-cells': [ gpio-controller ]
   gpio-line-names: ['#gpio-cells']
   ngpios: ['#gpio-cells']
   gpio-reserved-ranges: ['#gpio-cells']
```

Comment:
Probably should make all these also list gpio-controller which was indirectly before.